charts/drupal/templates/drupal-cron.yaml

```diff
@@ -98,24 +98,36 @@ spec:
                 exit "$exit_code"
 
             resources:
-              {{ if $job.resources }}
+              {{- if $job.resources }}
               {{- $job.resources | toYaml | nindent 14 }}
-              {{ else }}
+              {{- else }}
+              {{- if ($.Values.php.cronJobDefaults).resources }}
+              {{- $.Values.php.cronJobDefaults.resources | toYaml | nindent 14 }}
+              {{- else}}
               {{- $.Values.php.resources | toYaml | nindent 14 }}
               {{- end }}
+              {{- end }}
```

Comment:
Hmm, I was thinking that cronJobDefaults would be merged with job.resources. In a similar way how the logic for exposeDomainsDefaults works.

That would be more consistent with the way overriding chart values works overall.

Any objections or concerns if do it that way?

Reply:
I was now checking https://github.com/wunderio/frontend-project-k8s/pull/102 and noticed that it works exactly the way I meant. Let's change also this Drupal PR :)